### DISCUSSION TITLE
5V GND Availability

### DISCUSSION THREAD

**Discussion by MattGeale** [PiSupply/PiJuice#1004] [General]
Hey all,
I'm going to be honest, I'm a noob when it comes to this type of stuff, so please bear with me :)

I have a Pi Zero 2W and appropriate PiJuice hat... similar to a few other users, I want want to use the PiJuice to run a Pi in a instance where I can watch birds at night; as such, requiring IR lights. 

I've picked up a set of [these IR Lights](https://www.amazon.com/Infrared-Illuminator-Adjustable-Resistor-Raspberry/dp/B07FM6LL3V) (i.e. 2 total) and want to wire them up so that I can see in the dark with the Noir camera.

I've tried having a look at the hardware documentation for the PiJuice and the Ras Pi, but I'm getting myself a bit turned around. How can I do that with the PiJuice hat as the GPIO pins are obstructed? I'm following [this](https://electronics.stackexchange.com/questions/345178/connection-problem-of-night-vision-ir-led-for-raspi-camera) generic wiring diagram for two IR lights.

If anyone could give me a bit of guidance I'd be grateful :)

Thanks in advance!

**Comment by tvoverbeek** [comment]
Use a 40-pin GPIO extender with extra long pins. For example https://shop.pimoroni.com/products/2x20-pin-gpio-header-for-raspberry-pi-2-b-a?variant=1132812269
Put the PiJuice pHAT on the extender pins. Since it is a push through you will have all GPIO connections available, including 3.3V, 5V and ground.
